src/main/java/org/spongepowered/api/entity/living/player/Player.java

```diff
@@ -339,4 +343,42 @@ default GameModeData getGameModeData() {
      */
     void setSpectatorTarget(@Nullable Entity entity);
 
+    /**
+     * Checks if this player is in the boss bar.
+     *
+     * @param bar The boss bar
+     * @return {@code true} if the player is in the boss
+     *     bar, {@code false} otherwise
+     */
+    default boolean hasBossBar(final ServerBossBar bar) {
+        return this.getBossBars().contains(bar);
+    }
+
+    /**
+     * Adds this player to the boss bar.
+     *
+     * @param bar The boss bar
+     * @see ServerBossBar#addPlayer(Player)
+     */
+    default void addBossBar(final ServerBossBar bar) {
+        checkNotNull(bar, "bar").addPlayer(this);
+    }
+
+    /**
+     * Removes this player to the boss bar.
+     *
+     * @param bar The boss bar
+     * @see ServerBossBar#removePlayer(Player)
+     */
+    default void removeBossBar(final ServerBossBar bar) {
+        checkNotNull(bar, "bar").removePlayer(this);
+    }
+
+    /**
+     * Gets an unmodifiable collection of boss bars this player is currently in.
```

Comment:
`... that are shown to this player` or `... that apply to this player`. Sorry i should have proposed these directly, but I'm not sure about them either.